app/views/public_activity/webhook/_create.html.slim

```diff
@@ -0,0 +1,23 @@
+li
+  .activitie-container
```

Comment:
@tboerger it is correct. I don't recall exactly why it is like this, but it was done to fix an style issue. So, even if ugly, this is not something to be dealt in this PR :wink: